server/src/test/java/org/elasticsearch/index/mapper/CopyToMapperTests.java

```diff
@@ -635,4 +642,34 @@ public void testCopyFromMultiField() {
         assertThat(e.getMessage(),
             Matchers.containsString("[copy_to] may not be used to copy from a multi-field: [field.bar]"));
     }
+
+    public void testCopyToValueFetching() throws IOException {
```

Comment:
We could move this to `FieldFetcherTests`, so all field retrieval tests are in one place. It already has a case around `copy_to`.